src/panels/lovelace/components/hui-card-options.ts

```diff
@@ -127,12 +146,20 @@ export class HuiCardOptions extends hassLocalizeLitMixin(LitElement) {
     );
   }
 
-  private _moveCard(ev: MouseEvent): void {
-    const lovelace = this.lovelace!;
-    const path = this.path!;
-    lovelace.saveConfig(
-      moveCard(lovelace.config, path, [(ev.currentTarget! as any).index])
-    );
+  private _moveCard(): void {
+    const dialogShowEvent = "show-move-card-view";
+    const dialogTag = "hui-dialog-move-card-view";
+    fireEvent(this, "register-dialog", {
```

Comment:
Should this stuff be moved to a show-move-card-view-dialog.ts file?

Reply:
yes